### PR TITLE
fix: change seconds to duration

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:
Update flag description to duration instead of seconds.

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: controllers/util/util.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 )
 
 var (
-	SuccessDel = flag.Duration("job-cleanup-on-success-delay", 0, "Seconds to delay job deletion after successful runs. 0 means no delay")
-	ErrDel     = flag.Duration("job-cleanup-on-error-delay", time.Hour*24, "Seconds to delay job deletion after errored runs. 0 means no delay")
+	SuccessDel = flag.Duration("job-cleanup-on-success-delay", 0, "Duration to delay job deletion after successful runs. 0 means no delay")
```
Comment (sozercan) [inline controllers/util/util.go]: What is the default unit?

Comment (ashnamehrotra) [reply controllers/util/util.go]: By default time.Duration is nanoseconds if the user doesn't provide a unit with their input. I can add that to the description as well.

Comment (sozercan) [reply controllers/util/util.go]: can we default to seconds?

Comment (sozercan) [reply controllers/util/util.go]: this can be a follow up if you want

Comment (ashnamehrotra) [reply controllers/util/util.go]: we changed it from seconds to duration in #308 for consistency. Should we revert it back?

**Review by cpuguy83 — APPROVED** [review]

**Pull request merged**